### PR TITLE
Address concerns raised by @jricher about syntaxes.

### PR DESCRIPTION
**Pull request by msporny**
Does this PR address your concerns, @jricher?

### CODE REVIEW

**Comment by msporny** [comment]
Ping @jricher, @jonnycrunch -- does this summarize what we discussed on our call?

File: index.html [hunk 1]
```diff
@@ -2865,6 +2865,38 @@ <h2>Syntaxes</h2>
...
+serializing the <code>issuanceDate</code> value could be accomplished with a
+variety of date-time formats, the reality is that there is exactly one way to
+do so and the specification is explicit about this requirement.
+      </p>
+
+      <p>
+Syntaxes that provide intermediate expressions of the data model, one of which
+is detailed in Section <a href="#json-web-token"></a>, are allowed as long as
+the final syntactic expression that asserts conformance with this
+specification is truly conformant. Intermediate expressions are typically
```
Comment (dlongley) [inline index.html]: Saying something that asserts conformance needs to be "truly" conformant is odd/redunant. Let's simplify to: "are allowed as long as the final syntactic expression is conformant with this specification."

File: index.html [hunk 2]
```diff
@@ -2865,6 +2865,38 @@ <h2>Syntaxes</h2>
...
+documented in a way that explains how one transforms between the
+Verifiable Credentials Data Model and the intermediate expression.
+      </p>
+
+      <p>
+For the avoidance of doubt, implementers are able to determine if their
+intermediate expressions are loss-lessly compatible with the Verifiable
```
Comment (dlongley) [inline index.html]:
```suggestion
intermediate expressions are losslessly compatible with the Verifiable
```

File: index.html [hunk 3]
```diff
@@ -2865,6 +2865,38 @@ <h2>Syntaxes</h2>
...
+Credentials Data Model by converting their intermediate expression to JSON-LD
```
Comment (dlongley) [inline index.html]: "if they can convert their intermediate expression to a final expression in JSON-LD, compact that using a JSON-LD processor, and then convert the result back to the intermediate form without data loss."

File: index.html [hunk 4]
```diff
@@ -2865,6 +2865,38 @@ <h2>Syntaxes</h2>
...
+Syntaxes that provide intermediate expressions of the data model, one of which
+is detailed in Section <a href="#json-web-token"></a>, are allowed as long as
+the final syntactic expression that asserts conformance with this
+specification is truly conformant. Intermediate expressions are typically
+documented in a way that explains how one transforms between the
+Verifiable Credentials Data Model and the intermediate expression.
+      </p>
+
+      <p>
+For the avoidance of doubt, implementers are able to determine if their
```
Comment (dlongley) [inline index.html]:
```suggestion
For the avoidance of doubt, implementers are able to determine their
```

(In conjunction with the suggestion below).

File: index.html [hunk 5]
```diff
@@ -2865,6 +2865,38 @@ <h2>Syntaxes</h2>
...
+intermediate expressions are loss-lessly compatible with the Verifiable
+Credentials Data Model by converting their intermediate expression to JSON-LD
+using a JSON-LD processor and then back into their intermediate form without
+data loss. Note that this does not mean that the implemenation needs to use
+a JSON-LD processor in production systems but rather that a JSON-LD processor
+is one useful tool to use when attempting to determine conformance with
+intermediate expression formats and this specification.
```
Comment (jricher) [inline index.html]: I am not a fan of "intermediate expression" as a term, would propose "concrete serialization". 

We should have more concrete discussion about things like checking strings for @context and the like.

File: index.html [hunk 6]
```diff
@@ -3025,7 +3057,7 @@ <h4>Relation to the Verifiable Credentials Data Model</h4>
 use of specific JWT-registered <a>claim</a> names and specific JWS-registered
 header parameter names to allow systems based on JWT to comply with this
 specification. If these specific <a>claim</a> names and header parameters are
-present, their respective counterpart in the standard 
+present, their respective counterpart in the standard
```
Comment (jricher) [inline index.html]: This seems like a spurious change.

**Comment by jricher** [comment]
> I feel that the changes proposed in PR #539 are cleaner and result in a clearer distinction between the data model and the representative syntaxes than this PR does, but am not sure if that is the problem being addressed by either PR.

That was the core goal of #539 's text changes.

**Comment by msporny** [comment]
I'm closing this PR without merging in favor of #539. @jricher if there are any parts of this PR that you would like to save, add them to #539.

**Comment by msporny** [comment]
> finding it difficult to asses how well anyone's concerns might be addressed by the PR without having an issue to refer to.

This PR was an attempt to address concerns raised by @jricher on an offline call. He also believes #539 will address a number of issues raised by @nadalin.

**Pull request closed without merging**